examples/bench/run_bench.sh

```diff
@@ -1,18 +1,29 @@
 #!/bin/bash
 
 set -o errexit
-set -o nounset
```

Comment:
I don't think you need to remove this, probably because you're using the colon to specify what to use if the var is unset.